src/Controller/ReviewController.php

```diff
@@ -123,6 +123,7 @@ public function unfinalizedOrders(Request $request, OrderRepository $orderReposi
         $unlockedOrders = $orderRepository->getSiteUnlockedOrders($site);
         $unfinalizedOrders = $orderRepository->getSiteUnfinalizedOrders($site);
         $orders = array_merge($unlockedOrders, $unfinalizedOrders);
+        $this->reviewService->loadFirstFiveParticipantNames($orders);
```

Comment:
Would it be possible to use this method in the `today` method, also?

Reply:
Thought about it but the `getTodayParticipants` method returns a different array format I decided to use it only for other review tabs.